cockpit/values.yaml

```diff
@@ -193,6 +193,11 @@ api:
     periodSeconds: 30
     failureThreshold: 3
 
+  pdb:
+    enabled: false
+    minAvailable: ""
+    maxUnavailable: "50%"
```

Comment:
Do we need to set values if we disable pdb by default?

Reply:
Good question, I left it there for providing example...

Reply:
We could maybe put a comment with a link to the doc 🤷